clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/BlockingStatusChecker.groovy

```diff
@@ -74,6 +74,27 @@ class BlockingStatusChecker {
     result
   }
 
+  @PackageScope
+  void execute() {
```

Comment:
This method could be DRY'd up to be

void execute() {
  execute ( { return null } )
}